kolibri/plugins/learn/assets/src/vue/breadcrumbs/index.vue

```diff
@@ -16,6 +16,8 @@
         <span class="middle-breadcrumb landscape" v-for="crumb in topicCrumbs">
           <a v-link="topicLink(crumb.id)">{{ crumb.title }}</a>
         </span>
+        <!-- Had to add a space or else the breadcrumbs would jump.-->
+        <span class="middle-breadcrumb landscape"> {{' ' + title}}</span>
```

Comment:
![screen shot 2016-08-26 at 12 10 49 pm](https://cloud.githubusercontent.com/assets/5577205/18017402/2fd50684-6b86-11e6-8079-a538e9353673.png)
this design deliver both